simpledate/src/main/java/com/parassidhu/simpledate/SimpleDate.kt

```diff
@@ -72,6 +78,16 @@ fun Date?.toDateTimeStandardConciseIn12Hours(): String {
     return dateAsString(this, pattern)
 }
 
+fun Date?.toCompleteDate(): String {
+    val pattern = "yyyy-MM-dd'T'HH:mm:ss"
+    return dateAsString(this, pattern)
+}
+
+fun Date?.toCompleteDateTUC(): String {
+    val pattern = "yyyy-MM-dd'T'HH:mm:ss"
+    return dateAsString(this, pattern)
+}
+
```

Comment:
These two functions seem to be same. Also, I don't really think it will be significantly useful. So let's keep the rest of the 4 functions which are useful.

Reply:
Done